### PR TITLE
Caddy 2.4.5

### PR DESCRIPTION
**Pull request by hairyhenderson**
https://github.com/caddyserver/caddy/releases/tag/v2.4.5

This also bumps to xcaddy 0.2.0, Alpine 3.14, and Go 1.17!

Closes #208, closes #202, closes #203, and closes #205 

Signed-off-by: Dave Henderson <dhenderson@gmail.com>

### CODE REVIEW

File: stackbrew-config.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+      amd64: 1a19a8d94c6c16d8e671c4dad1e835e7d0d586a36e15f5300b374e5d31cf10eafdf07bc2c24d5589cdfaff1dd276e4627917802e858e14dc2351797d7dec7586
+      arm32v6: 5b68e8b9491507591bfa42880a771f5c687e9e081ccc249a2356b0a1cafc108214e6915d86a5342e6eb99c329942cd027bfd8960d38dae11c91a47a43926b1a8
+      arm32v7: e39210f8e3634ebc85a3a32bda4bcb2263d508744f8ba3b2e9e56012cda17cabe16f2318f991e3d1d8306da49fcc2fa3bfef98f3412bbcf734376ea3bcbdf2ea
+      arm64v8: da23940ff938c953d288555366e09632b31fd248e337da1990cebf0471957d14196f38396fd055d4b409f9960e2d8fb94b26ab435662a871b5ea300b9e77ad25
+      ppc64le: cf8d73a238628cfacea7d309f64248a6e25c76dd76a669a646845b4ca729fd677e30a2a29c75936d810b9f55df89fa17f9b6bc42c5d69e8e02b12ef4e334ad91
+      s390x: 21631de0d9c93346a9ab8658990c87bacda9c401ddb4b850a59f0c9f9665e0b93b02c0516df55eb03829f0eb5fb546683ba2d5e7b8f6174a7dfb3d13db9d81cd
+      windows_amd64: 794fed88b38bfeb6b3b84d63f0887c4534fc3e94bd31173182b5af80fc7783beb41192bccffdb257a9bdf59541c6822a75fdbe714d65abc3c0cce9c02018eb82
 xcaddy_config:
   version: '0.1.9'
```
Comment (francislavoie) [inline stackbrew-config.yaml:15]: Maybe we should wait for a new tag of xcaddy to go along with this /cc @Mohammed90 @mholt (Matt has to be the one to do the tagging, we check the signature of the tag on the xcaddy repo)

Comment (mholt) [reply stackbrew-config.yaml:15]: v0.2.0 is releasing now

Comment (francislavoie) [reply stackbrew-config.yaml:15]:
```
    amd64: b02fef09e1ea4ec26d87f1d24a442aaf247ca79b391064d7708e080e4bc5e14204381a3c573122b8b5f10de4495cf4e54fd7b90122ed45127673284d9619a0b8
    arm32v6: 735f5327e47d7a4fbc224b305231f211475b26f7a0def11eccabc15c98fc1864f36370de235869a093a3c97a7ebe1ca255a4e64ab4eaf77dae89bd8a28a708a0
    arm32v7: d013d38f62ca548dcdea532224cd8571975d2b8c9bfcf98fc392907fd97d0ecfd0516d457b32988ef7215c626d51df2c2b6ca17f7c3cc14db656b84d4b1b8304
    arm64v8: d64fbed9556bf3997951d5337b91eb85b2a3abf3bf6d3e28a838f1977c276e39087e435b6def9a860cdd6d1f0095602374ddedb3e6c382b8a29c7d446623f651
    ppc64le: b304e6129e4928155c1c9eda33b76b43024ba34bd0197445a8dff12b45472f92d96d90b9083525b480d796d94bfe861cc415b4dbaad56cfcc5e6f1c366cc476c
    s390x: ad3b5e27804d77f76429c98babf8df6f3418b613ec94370afc57dab5ed7f9b6492118f6706b01d73f1ff352c3f5618f41cb87dfbc3b089b2230d064bd8153a41
windows_amd64: 233a57384b1f82e9420567da74b4fbd19e898112e43b8447dbdb8ddde15cb4d8a66aea58307ccdda74d37c5e525f0dc563f83d4670aee048842754eee9a3bc2b
```

Comment (hairyhenderson) [reply stackbrew-config.yaml:15]: nice, thanks!

**Review by francislavoie — APPROVED** [review]

**Pull request merged**